docs/source/dev_guide/algorithms/svm/support-vector-machine-classifier.rst

```diff
@@ -43,14 +43,18 @@ belongs, the problem is to build a two-class Support Vector Machine
 
 Training Stage
 --------------
-The SVM model is trained using the Boser method [Boser92]_
-reduced to the solution of the quadratic optimization problem
+
+|short_name| provides two methods to train the SVM model:
+
+- Boser method [Boser92]_ that is the Sequential Minimal Optimization algorithm proposed by Platt [Platt98]_
```

Comment:
@PetrovKP , add details about differences between Boser and Platt. According to your formulation, they are the same, but it isn't true. Boser in '92 proposed something different then Platt in '98.

Reply:
```suggestion
- Boser method [Boser92]_ - performance-oriented variant of Boser [Boser92] and Platt [Platt98] algorithms.
```